content/plugins/provide-plugin.md

```diff
@@ -3,13 +3,16 @@ title: ProvidePlugin
 contributors:
   - sokra
   - simon04
+  - re-fort
 ---
 
 ```javascript
 new webpack.ProvidePlugin({identifier1: 'module1', /* ... */})
+// or
+new webpack.ProvidePlugin({identifier1: ['module1', 'property1'], /* ... */})
 ```
 
-Automatically loads modules. Whenever the `identifier` is encountered as free variable in a module, the `module` is loaded automatically and the `identifier` is filled with the exports of the loaded `module`.
+Automatically loads modules. Whenever the `identifier` is encountered as free variable in a module, the `module` is loaded automatically and the `identifier` is filled with the exports of the loaded `module`(of `property`).
```

Comment:
s/of property/of property in order to support named exports/ (give the reader a clue why this is useful)